dialogue-clients/src/main/java/com/palantir/dialogue/clients/AugmentClientConfig.java

```diff
@@ -93,4 +90,26 @@ static ClientConfiguration getClientConf(ServiceConfiguration serviceConfig, Aug
 
         return builder.build();
     }
+
+    private static TrustContextFactory buildTrustContextFactory(AugmentClientConfig augment) {
```

Comment:
perhaps `buildTrustContextFactory(Optional<Provider> augment)` to make it clear what data we use from the AugmentClientConfig?

Reply:
Handled in https://github.com/palantir/dialogue/pull/1923/commits/29d64ac261593516cfbdb84d4c30cf528f984ddb